src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 1px 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+  padding: 10px 0;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  list-style: none;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+}
+
+.nav__link {
+  position: relative;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+}
+
+.hover {
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  left: 0;
+  bottom: -23px;
+  height: 4px;
+  width: 100%;
+  background-color: #00acdc;
+}
+
+.nav__link:active::after {
```

Comment:
this is redundant .is-active class is enough

Reply:
when I remove the pseudo element ::after everything breaks for me, therefore he must be!
![image](https://user-images.githubusercontent.com/114721219/198865476-0bfb9221-7fc4-4780-a677-08ac4a25fe6c.png)

Reply:
remove `active` pseudoclass, it should be default!